src/main/resources/zenduty-template.peb

```diff
@@ -1,22 +1,15 @@
 {
-    {% if title is defined %}
-        "title": "{{title}}",
+    {% if message is defined %}
+        "message": "{{message}}",
+    {% endif %}
+    "alert_type": "{% if alertType is defined %}{{alertType}}{% elseif firstFailed == false %}resolved{% else %}error{% endif %}",
+    "entity_id": "{% if entityId is defined %}{{entityId}}{% else %}{{execution.id}}{% endif %}",
+    {% if urls is defined %}
+        "urls": [
+              {% for url in urls %}
+                      {{url | json}},
```

Comment:
```suggestion
                      {{url}},
```

I don't understand why you add the json filter here as URLs are string in the properties